bin/test

```diff
@@ -59,29 +59,7 @@ iexCurrentVersion="$(iex -v | tail -n 1 | cut -d " " -f2)"
 echo "[DEBUG] Current Elixir Version: $iexCurrentVersion"
 otpCurrentVersion="$(erl -eval 'erlang:display(erlang:system_info(otp_release)), halt().' -noshell | sed 's/\"//g')"
 echo "[DEBUG] Current OTP Version: $otpCurrentVersion"
-requiredMaxVersion="1.12.0"
```

Comment:
Any reason we only removed support/directories for Phx 1_3 and 1_4, but not the 1_5 stuff as well?

Reply:
1_5 became 1_6

Reply:
I'm not sure what you mean by that?  Did you just rename the 1_5 directory to 1_6 and search/replace the strings to 1.6?  If so, that won't work.  We need to spin up and init a new app on Phoenix 1.6 to get a "stock/baseline" 1.6 app in place.